lib/warder.rb

```diff
@@ -11,7 +11,8 @@
 require 'warder/sandi_rules_runner'
 require 'warder/bundle_audit_runner'
 require 'warder/coffee_lint_runner'
-require 'warder/cli/arguments'
+require 'warder/aruba_app'
+require 'warder/arguments'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.